examples/classification/main.py

```diff
@@ -184,8 +184,9 @@ def main_worker(current_gpu, config: SampleConfig):
     if config.execution_mode != ExecutionMode.CPU_ONLY:
         cudnn.benchmark = True
 
+    print_statistics(compression_ctrl.statistics())
```

Comment:
please do it for other samples as well

Reply:
don't forget about staged_worker

Reply:
but before this, please add a check that this method is called in the sanity tests (on train and eval mode, not called on export to onnx)
should be straightforward with mocker.patch:

```
mocked_printing = mocker.patch('examples.common.utils.print_statistics')
mocked_printing.assert_called_once_with()
```